staging/src/k8s.io/apiserver/pkg/endpoints/handlers/response.go

```diff
@@ -204,20 +216,20 @@ func (e *watchEncoder) doEncode(obj runtime.Object, event watch.Event, w io.Writ
 }
 
 type watchEncoderIdentifier struct {
-	Name            string `json:"name,omitempty"`
-	EmbeddedEncoder string `json:"embeddedEncoder,omitempty"`
-	Encoder         string `json:"encoder,omitempty"`
-	EventType       string `json:"eventType,omitempty"`
+	Name                string `json:"name,omitempty"`
+	WatchObjectsEncoder string `json:"watchObjectsEncoder,omitempty"`
```

Comment:
please don't rename fields

Reply:
why? the struct is not persisted.

Reply:
It was supposed to be aligned with the my other comment to avoid renaming in general.